stories/AxisYWithContainer.stories.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useRef } from 'react';
+
+import Resizable from '../src/Resizable';
+import SampleBox from './components/SampleBox';
+import SampleSeparator from './components/SampleSeparator';
+
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+
+export default {
+  title: 'main/Resizable',
+  component: Resizable,
+} as ComponentMeta<typeof Resizable>;
+
+const Template: ComponentStory<typeof Resizable> = (props) => {
+  const containerRef = useRef<HTMLDivElement>(null);
+
+  return (
+    <>
+      <div style={{ padding: '16px', background: 'red' }} />
+      <div ref={containerRef}>
+        <Resizable {...props} containerRef={containerRef}>
+          {({ position: y, separatorProps }) => (
+            <div
+              style={{
+                display: 'flex',
+                flexDirection: 'column',
+                height: '100vh',
+                overflow: 'hidden',
+              }}
+            >
```

Comment:
@KlonD90 
I know it's a small detail, but can I ask you to make sure the element doesn't overflow?
```suggestion
            <div
              style={{
                display: 'flex',
                flexDirection: 'column',
                height: 'calc(100vh - 32px)',
                overflow: 'hidden',
              }}
            >
```